ci/run_tests.go

```diff
@@ -109,6 +110,11 @@ func recordResults(logger *zap.SugaredLogger, db database.Database, rData *RunDa
 		approvedStatus = pb.Submission_APPROVED
 	}
 
+	// Don't change the build date on a manual rebuild, the date is used as a submission's delivery date
```

Comment:
Note that the `recordResults` method has been updated somewhat in #473, so it would be good if this PR didn't change things too much beyond what is already here...

Reply:
Yes, will make sure it is compatible with #473.

Reply:
Updated.